src/format/FileMetaData.php

```diff
@@ -207,6 +209,57 @@ public function getName()
     }
 
 
+  private function convertAssocToKeyValues(array $customMeta): array{
```

Comment:
This is an auto-generated file from the Thrift sources. Please don't modify, as it will be overwritten during a re-generation.